build-tools/packages/build-cli/docs/codeCoverageDetails.md

```diff
@@ -91,3 +91,16 @@ The code coverage PR checks will fail in the following cases:
 - If the code coverage(line and branch) for a newly added package is < 50%.
 
 The enforcement code is in the function `isCodeCoverageCriteriaPassed`.
+
+### What to do if the code coverage check fails:
+
+- In the code coverage summary, a checkbox will appear which will not allow the code coverage check to fail the build.
```

Comment:
Did we consider doing this at the pipeline level? Have a flag in the pipeline, make the step fail or succeed (or warn, ideally) depending on the flag. The approach you took certainly feels easier to use, because the alternative would require a manual pipeline run and doesn't persist, like updating the comment and ticking the box does.

Reply:
My reason of doing it this way is because I first want to let the code coverage check run with build failure (in case of regression), rather than allowing someone to skip this check without actually knowing what is the regression the PR is causing and then the dev can actually give it a thought what to do about it or how to reason about it and take the decision appropriately and then it is also visible on the PR what we are actually doing and yeah it has the benefits that you noted as well.